doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -79,7 +79,9 @@ Bluetooth® LE
 Bluetooth mesh
 --------------
 
-|no_changes_yet_note|
+* Updated:
+
+  * :ref:`bt_mesh_dm_srv_readme` and `bt_mesh_dm_cli_readme` model IDs and opcodes have been updated to avoid clash with Simple OnOff Server and Client models.
```

Comment:
```suggestion
  * :ref:`bt_mesh_dm_srv_readme` and :ref:`bt_mesh_dm_cli_readme` model IDs and opcodes have been updated to avoid conflict with Simple OnOff Server and Client models.
```